Sources/SwiftLayout/Refactoring/Protocol/LayoutTraversal.swift

```diff
@@ -0,0 +1,201 @@
+import Foundation
+import UIKit
+
+typealias TraverseHandler = (_ superview: UIView?, _ subview: UIView, _ identifier: String?, _ animationDisabled: Bool) -> Void
+typealias ConstraintHandler = (_ superview: UIView?, _ subview: UIView, _ constraints: [Constraint], _ viewInfoSet: ViewInformationSet) -> Void
+
+protocol LayoutTraversal {
```

Comment:
I think LayoutTracersal is an essential element of Layout
So, rather than separate the Layout protocol and the LayoutTraversal protocol,
What do you think of defining traverse(....) methods as requirements of the Layout protocol?

Reply:
> I think LayoutTracersal is an essential element of Layout
> So, rather than separate the Layout protocol and the LayoutTraversal protocol,
> What do you think of defining traverse(....) methods as requirements of the Layout protocol?

oh you pick right point.